examples/Benchmarking RDF against MDAnalysis.ipynb

```diff
@@ -32,8 +32,8 @@
    "outputs": [],
```

Comment:
<p>Minor nitpick, but these timings would make more sense sequentially if you moved the last one to the top: serial vs MDAnalysis, parallel vs serial, then parallel vs MDAnalysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>